Oneirocom/Magick
# New AIDE left sidebar

## What Changed:

I have changed the design for the AIDE, I added the following to the AIDE:

- Left side bar container
- Agent dropdown and
- File tree drawer

## How to test:

- Navigate to the `left_sidebar` branch after pulling the latest changes and check the home page UI to test it

## Additional information:


https://github.com/Oneirocom/Magick/assets/55635977/1031c67e-0a36-4bbf-953f-4a47577682fe
